README.md

```diff
@@ -1,70 +1,226 @@
+- [Live Demo Link](https://explore-the-world-countries.netlify.app/)
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+ https://loom.com/share/fe39b7534cec4cdd86d7c30ed409f5d9
 
-### `npm run build` fails to minify
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+- Create a local directory that you want to clone the repository.
+
+- Open the command prompt in the created directory.
+
+- On the terminal run this command git clone https://github.com/Osoro254Alex/Bookstore
+
+- Go to the repository folder using command prompt cd Bookstore.
+
+- Install the dev dependencies for linters run npm install.
+
+### Prerequisites
+
+- Basic Knowledge of HTML & CSS
+- Basic Knowledge of [JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript) & [ES6](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/lessons/lesson_es6-what_is_it_about.md) syntax
+- Basic Knowledge of [React & Redux Setup](https://reactjs.org/docs/getting-started.html) setup
+
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+To make use of this project, kindly run any of the following commands.
+
+```
+git clone https://github.com/Osoro254Alex/Bookstore.git
+```
+
+or
+
+```
+git clone git@github.com:Osoro254Alex/Bookstore.git
+```
+## **INSTALL**
+```
+npm install
+```
+```
+## **Run On Server**
+To run the project on server, run below command
+
+```
+npm start
+```
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+
+👤 **Alex Ogeto**
+
+- GitHub: [@alex_ogeto](https://github.com/Osoro254Alex)
+- LinkedIN:[@Osoro254Alex](https://www.linkedin.com/feed/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 **About Me**
+
+I'm full stack software developer, Computer science, and Microverse student.
+**Stack:** JavaScript, Bootstrap, Ruby, Rails, React, Redux. Available for hire!
+
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Osoro254Alex/Capstone3-World-Countries-/issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- [Microverse](https://www.microverse.org/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
```

Comment:
- [ ] Kindly include an acknowledgment for Nelson Sakwa according to the Creative Commons License requirement.